coffeelint.json

```diff
@@ -0,0 +1,5 @@
+{
+  "max_line_length": {
+    "level": "ignore"
```

Comment:
Yeah, I agree. However, this linter has a couple of large lines and I did not want to add the specs and change the code at once. Would it be cool if I send another PR after this one gets merged to fix this?